src/operator/nn/fully_connected.cc

```diff
@@ -316,11 +316,9 @@ NNVM_REGISTER_OP(_backward_FullyConnected)
   const FullyConnectedParam& params = nnvm::get<FullyConnectedParam>(attrs.parsed);
   return params.no_bias ? 2 : 3;
 })
-#if MXNET_USE_MKLDNN == 1
```

Comment:
Why are we removing this condition?

Reply:
because we need additional temp space now.